flow/genericactors.actor.h

```diff
@@ -1101,30 +1101,6 @@ Future<T> tagError( Future<Void> future, Error e) {
 	throw e;
 }
 
-//If the future is ready, yields and returns. Otherwise, returns when future is set.
```

Comment:
These functions might be useful to someone in the future even if they are not used right now